python/ray/actor.py

```diff
@@ -1306,6 +1309,9 @@ def __init__(
             actor_creation_function_descriptor
         )
         self._ray_function_descriptor = {}
+        # This is incremented each time `bind()` is called on the actor method
```

Comment:
```suggestion
        # This is incremented each time `bind()` is called on an actor handle
```

Reply:
Hmm one thing I just realized is that this may not work properly if the caller has two different copies of the same actor handle. Not a common case but it can happen. Could you put a TODO here to handle that case?

Reply:
Done.